### PR TITLE
Balance military unit production power costs

### LINKED ISSUE #824: Air is too dominant
Air is too dominant during matches: the Starports cost too low power compared to other military production buildings.

What about changing Starport power requirement to 30, Vehicle Factory to 120 and Harbor to 90?

### PR DESCRIPTION
**Pull request by WilliamWeberBerrutti**
This PR attempts to fix #824.

### CODE REVIEW

File: mods/hv/rules/buildings.yaml [hunk 1]
```diff
@@ -683,7 +683,7 @@ STARPORT:
 		PrimaryCondition: primary
 		SelectionNotification: PrimaryBuildingSelected
 	Power:
-		Amount: -10
+		Amount: -30
```
Comment (Dzierzan) [inline mods/hv/rules/buildings.yaml:686]: Make it `-120`.

File: mods/hv/rules/buildings.yaml [hunk 2]
```diff
@@ -752,7 +752,7 @@ FACTORY3:
 	RevealsShroud@Hacked:
 		Range: 2c0
 	Power:
-		Amount: -150
+		Amount: -120
```
Comment (Dzierzan) [inline mods/hv/rules/buildings.yaml:755]: Keep it `-150`.

File: mods/hv/rules/buildings.yaml [hunk 3]
```diff
@@ -1795,7 +1795,7 @@ HARBOR:
 	ProductionBar:
 		ProductionType: Ship
 	Power:
-		Amount: -150
+		Amount: -90
```
Comment (Dzierzan) [inline mods/hv/rules/buildings.yaml:1798]: Make it `-120`.

**Review by Dzierzan — CHANGES_REQUESTED** [review]

File: mods/hv/rules/buildings.yaml (hunk 1 above)
Comment (Mailaender) [reply mods/hv/rules/buildings.yaml:686]: Maybe even more. The `-10` is definitely a mistake as you only need one factory, not multiple air fields.

**Comment by Dzierzan** [comment]
LGTM, squash commits and make it ready for merging.

**Pull request merged**